airsonic-main/src/main/webapp/WEB-INF/jsp/externalPlayer.jsp

```diff
@@ -4,68 +4,17 @@
 <html>
 <head>
     <%@ include file="head.jsp" %>
-    <script type="text/javascript" src="<c:url value="/script/swfobject.js"/>"></script>
     <script type="text/javascript" src="<c:url value="/script/prototype.js"/>"></script>
-
-    <meta name="og:type" content="album"/>
```

Comment:
Any reason for removing this particular meta tag?

Reply:
Nope, I'll add it back.